tgui/packages/tgui/interfaces/common/AtmosControls.js

```diff
@@ -176,3 +176,64 @@ export const Scrubber = (props, context) => {
     </Section>
   );
 };
+
+export const dangerMap = {
+  0: {
+    color: 'good',
+    localStatusText: 'Optimal',
+  },
+  1: {
+    color: 'average',
+    localStatusText: 'Caution',
+  },
+  2: {
+    color: 'bad',
+    localStatusText: 'Danger (Internals Required)',
+  },
+};
```

Comment:
Why isn't this just an array?

Reply:
To just more clearly define indexes, purely for readability.